app/models/ApprovalProcessReview.scala

```diff
@@ -18,16 +18,52 @@ package models
 
 import java.time.{LocalDate, ZonedDateTime}
 import java.util.UUID
+import play.api.libs.json._
+import play.api.libs.functional.syntax._
+import core.models.MongoDateTimeFormats.Implicits._
 
 case class ApprovalProcessReview(
-    id: UUID,
-    ocelotId: String,
-    version: Int,
-    reviewType: String,
-    title: String,
-    pages: List[ApprovalProcessPageReview],
-    lastUpdated: LocalDate = LocalDate.now(),
-    result: String = "",
-    completionDate: Option[ZonedDateTime] = Option.empty[ZonedDateTime],
-    completionUser: Option[String] = Option.empty[String]
+  id: UUID,
+  ocelotId: String,
+  version: Int,
```

Comment:
If OcelotVersion is classed as Optional in ApprovalProcessMeta should this also be Optional ?